pkg/cfn/manager/api.go

```diff
@@ -36,9 +36,10 @@ type ChangeSet = cloudformation.DescribeChangeSetOutput
 
 // StackCollection stores the CloudFormation stack information
 type StackCollection struct {
-	cfn  cloudformationiface.CloudFormationAPI
-	spec *api.ClusterConfig
-	tags []*cloudformation.Tag
+	cfn         cloudformationiface.CloudFormationAPI
+	waitTimeout time.Duration
+	spec        *api.ClusterConfig
```

Comment:
We still keep a pointer here, but it's arguably more sensible, as there aren't many different kinds of calls to stack manager from the outside, so it's sensible to set it once.